### ISSUE TITLE
Add Gauntlet as a tournament type

### ISSUE THREAD

**Issue by gahtan-syarif** [Disservin/fastchess#492]
reopening this in case youre interested in adding it

**Comment by Carbecq** [comment]
I add my vote for this enhancement. The only possibility is cutechess, but is has time problems.

**Comment by Disservin** [comment]
Yeah we are a bit closer to it now, need to add the code for the opening book policy and add a new tournament type

**Comment by Carbecq** [comment]
Hi,



Great news) Many thanks



Philippe



De : disservin ***@***.***>
À : Disservin/fastchess ***@***.***>
Sujet : Re: [Disservin/fastchess] Add Gauntlet as a tournament type (Issue #492)
Date : 21/10/2024 12:07:35 Europe/Paris
Copie à : Carbecq ***@***.***>;
   Comment ***@***.***>


Yeah we are a bit closer to it now, need to add the code for the opening book policy and add a new tournament type

**Comment by Matthies** [comment]
Is anybody working on this? If not I would give it a try, but don't hold your breath.

**Comment by Disservin** [comment]
Not at the moment to my knowledge, I'm not sure how extensible the current logic for the different tournament types is at the moment since I only bothered with round robin so might need some rewrite 🙃 if you have problems feel free to message me
